models/aether_lat-lon/readme.rst

```diff
@@ -0,0 +1,191 @@
+Aether Rectangular Grid Interface
+=================================
+
+Overview
+--------
+
+The `Aether`_ ("eether") space weather model can be implemented 
+on a logically rectangular grid "lat-lon", 
+or on an the cubed-sphere grid (see ../aether_cubed_shere).
+This is the interface to the lat-lon version.
+
+.. _Aether: https://aetherdocumentation.readthedocs.io/en/latest/
```

Comment:
```suggestion
Aether is available on GitHub https://github.com/AetherModel/Aether
```

Reply:
What's the advantage of referring to the model on github, which is more focused on code,
rather than the model documentation?

Reply:
I don't think the readthedocs site is aether right? It talks about installing CAM, CESM and WACMX.
Seems safest to point people at the Aether github repo.

Reply:
The original link points to the "Aether Model Documentation" page.
The next page is the Aether github instructions.  After that are system requirements for Aether.
Then it gets into the CAM and WACCM pieces which were useful for building Aether.
Whichever you think is best.

Reply:
lets point people direct to Aether.